examples/webgl_loader_mmd.html

```diff
@@ -160,23 +172,76 @@
 
 				if( mesh ) {
 
-					mixer.update( clock.getDelta() );
-
-					/* Note: ikSolver.resetLinks() is workaround for r74dev.
-					 *       THREE.PropertyMixer.apply() seems to have buffer cache
-					 *       when mixer.update() is called.
-					 *       ikSolver.update() change bone position/quaternion
-					 *       without mixer.update() then buffer cache will be inconsistent.
-					 *       ikSolver.resetLinks() explicitly resets IK link bones quaternion and
-					 *       tries to avoid issues caused by this inconsistent.
-					 */
-					ikSolver.resetLinks();
+					var delta = clock.getDelta();
+					mixer.update( delta );
+
+					// see below for what this's for
+					backupBones( mesh );
+
 					ikSolver.update();
+					physics.update( delta );
 
 				}
 
 				renderer.render( scene, camera );
 
+				if ( mesh ) {
+
+					// see below for what this's for
+					restoreBones( mesh );
+
+				}
+
+			}
+
+			/*
+			 * Note: These following three functions are workaround for r74dev.
+			 *       THREE.PropertyMixer.apply() seems to save values into buffer cache
```

Comment:
@takahirox 
No code. I tried sketch out what needs to be done above and had hoped you'd join in completing the picture.

```javascript
/**
 * @param {!THREE.AnimationPlugin} plugin
 * @param {THREE.Object3D|THREE.AnimationObjectGroup=} opt_localRoot
 */
AnimationMixer.prototype.pluginAction = function(plugin, opt_localRoot) { ... }; 
// treat plugin as an _Action, this way it can be started / stopped 
// for individual objects, has its own time scale, etc.

/** @param {!string} path */
AnimationMixer._PluginAction.prototype.bindProperty_ = function(path) { ... };

/** @type {!Array<!PropertyMixer>} */
AnimationMixer._Action.prototype.propertyBindings_;

/** @param {!THREE.AnimationMixer._PluginAction} action */
AnimationPlugin.prototype.bindObject = function(action) { ... };
// called when the plugin is started on an object
// call action.bindProperty_ to get the PropertyMixers to work on 

/**
 * @param {!THREE.AnimationMixer._PluginAction} action
 * @param {!number} deltaTime
 * @param {!number} accuIndex accu region used for the current frame (0 or 1)
 */
AnimationPlugin.prototype.perform = function(action, deltaTime, accuIndex) { ... };
// called every frame, update the the PropertyMixers' current accu region here 

```

Plus some renaming to expose the mixing buffers of `PropertyMixer`.

That interface work for all plugins?